File: loan-decisioning-api/src/foam.py
import foam
from src.config import FOAM_API_KEY, IS_PRODUCTION

foam.init(
    service_name="loan-decisioning-api",
    is_production=IS_PRODUCTION,
    api_key=FOAM_API_KEY,
    auto_instrument=True,
    trace_db_queries=True,
    trace_http_clients=True,
)

File: merchant-integration-sdk/src/foam.ts
import * as foam from '@foam-ai/browser-opentelemetry';
import { FOAM_API_KEY, IS_PRODUCTION_ENV } from './config';

foam.init({
  serviceName: 'merchant-integration-sdk',
  isProduction: IS_PRODUCTION_ENV,
  apiKey: FOAM_API_KEY,
  captureErrors: true,
  capturePerformance: true,
  sampleRate: 1.0,
});